site/code-snippets/web5/build/decentralized-web-nodes/query-from-dwn.js

```diff
@@ -63,3 +63,51 @@ export async function queryRecordWithParentId(web5) {
 
   return response;
 }
+
+export async function queryFromDwnByPathAndStructure(web5) {
+
+  const musicProtocolDefinition = {
+    protocol: "https://music.org/protocol",
+    types: {
+      audio: {
+        schema: "https://schema.org/Playlist",
```

Comment:
```suggestion
        schema: "https://schema.org/AudioObject",
```

if you'd like to use a resolvable schema, can use https://schema.org/AudioObject